vim.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:
```

Comment:
this file should not be included seems like

Reply:
removed